fastlane/metadata/ar-SA/keywords.txt

```diff
@@ -1 +1 @@
-اجتماعي، شبكة، ملاحظات، jetpack، صور، كتابة، تحديد المواقع جغرافيًّا، وسائط، مدونة، ووردبريس، موقع و
\ No newline at end of file
+اجتماعي، شبكة، ملاحظات، jetpack، صور، كتابة، تحديد المواقع جغرافيًّا، وسائط، مدونة، ووردبريس، موقع ويب، تدوين، تصميم
```

Comment:
Why did keywords change this time around? They rarely change so I doubt someone decided to change them between beta 1 and beta 2?